index.html

```diff
@@ -27,17 +27,42 @@
       </p>
     </section>
     <section id="sotd">
-        <p>This is a draft for a future SocialCG report.</p>
+        <p>This is a draft of a future SocialCG report.</p>
       </section>
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-        Data portability is the ability of a user to move their data on one
+        Data portability is the ability of a user to move their data from one
         system to another system. This includes moving between ActivityPub
         implementations, moving to different installations of the same
         software, or changing domains.
       </p>
+
+    <section class="informative">
+      <h2>Terminology</h2>
+
+      <p>
+        The following terms are used to describe concepts in this report.
+      </p>
+
+      <dl class="termlist">
+        <dt><dfn data-lt="instance|instances">instance</dfn></dt>
+        <dd>An informal name for an ActivityPub <a>server</a>, specifically,
+          a particular instance of a given ActivityPub server implementation.
+          In current implementations, instances are typically bound to a given
+          URL / Web domain, which has implications for identity portability and
+          data portability in general.
+        </dd>
+        <dt><dfn data-lt="server|servers">server</dfn></dt>
+        <dd>
+          An implementation of an <strong>ActivityPub conformant Server</strong>,
```

Comment:
```suggestion
          A piece of software that, when deployed, can function as <strong>ActivityPub-conformant Server</strong>,
```

Reply:
I'm not sure 'piece of software' sounds quite right..